### ISSUE TITLE
value_affect_tracing_table_test is flaky

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla#9163]
The test was introduced in https://github.com/scylladb/scylla/issues/7265
It fails from time time as follows:

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/873/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
```
Error: probability=0.6 actual=0.63 diff=0.030000000000000027 allowed=0.03
```

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/874/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
```
Error: probability=0.001 actual=0.0018 diff=0.0007999999999999999 allowed=0.00075
```

https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/896/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
```
Error: probability=0.6 actual=0.638 diff=0.038000000000000034 allowed=0.03
```

I'm not sure what the original tolerance values were based on, but it seems the actual differences we see are slightly larger.

@roydahan can you please assign someone to look into this?

**Comment by roydahan** [comment]
@fgelcer can you please check if we need to fix the test?
It was added as part of https://github.com/scylladb/scylla-dtest/pull/1750

**Comment by fgelcer** [comment]
> @fgelcer can you please check if we need to fix the test?
> It was added as part of [scylladb/scylla-dtest#1750](https://github.com/scylladb/scylla-dtest/pull/1750)

checking it now

**Comment by fgelcer** [comment]
i'm just able to reproduce it locally, i'm trying to understand if the test is not good, or if there is some issue with Scylla...

**Comment by fgelcer** [comment]
@bhalevy , does it happen only for master?

**Comment by bhalevy** [comment]
> @bhalevy , does it happen only for master?

@fgelcer no, I saw it also in
https://jenkins.scylladb.com/view/scylla-4.5/job/scylla-4.5/job/dtest-release/44/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
```
Error: probability=0.6 actual=0.631 diff=0.031000000000000028 allowed=0.03
```

https://jenkins.scylladb.com/view/scylla-4.5/job/scylla-4.5/job/dtest-release/28/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
```
Error: probability=0.6 actual=0.568 diff=0.03200000000000003 allowed=0.03
```

And

https://jenkins.scylladb.com/view/scylla-4.4/job/scylla-4.4/job/dtest-release/80/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
```
Error: probability=0.6 actual=0.637 diff=0.03700000000000003 allowed=0.03
```

**Comment by fgelcer** [comment]
> > @bhalevy , does it happen only for master?
> 
> @fgelcer no, I saw it also in
> https://jenkins.scylladb.com/view/scylla-4.5/job/scylla-4.5/job/dtest-release/44/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
> 
> ```
> Error: probability=0.6 actual=0.631 diff=0.031000000000000028 allowed=0.03
> ```
> 
> https://jenkins.scylladb.com/view/scylla-4.5/job/scylla-4.5/job/dtest-release/28/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
> 
> ```
> Error: probability=0.6 actual=0.568 diff=0.03200000000000003 allowed=0.03
> ```
> 
> And
> 
> https://jenkins.scylladb.com/view/scylla-4.4/job/scylla-4.4/job/dtest-release/80/testReport/junit/nodetool_additional_test/TestGetTraceProbability/value_affect_tracing_table_test/
> 
> ```
> Error: probability=0.6 actual=0.637 diff=0.03700000000000003 allowed=0.03
> ```

in the fix i will add the back port labels to 4.5 and 4.4
